Shared (App)/Models/Contract.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Contract.swift
+//  Wallet (iOS)
+//
+//  Created by Stefano on 05.12.21.
+//
+
+struct Contract {
+    let address: String
+    let name: String
+    let abi: String
+    let nameTag: String? // Could be an array of name tags
+}
+
+struct ContractInfo {
```

Comment:
ContractInfo is currently used for the plist name. May want to consider a better model once we also have the contract tx parsing.